gobrew.go

```diff
@@ -705,8 +713,20 @@ func (gb *GoBrew) getGithubTags(repo string) (result []string) {
 	var tags []Tag
 
 	if err := json.Unmarshal(data, &tags); err != nil {
-		utils.Errorf("[Error] Rate limit exceed")
-		os.Exit(2)
+		utils.Errorln("[Error] Github tags Rate limit exceeded")
+		utils.Infoln("[Info] Trying with local cache")
+		if err := json.Unmarshal(cachedData, &tags); err != nil {
```

Comment:
On https://github.com/kevincobain2000/gobrew/actions/runs/5036076816/jobs/9031983134 we got error on getting json from network. Not Unmarshal these data.

I think we should use cached data on line, if we got error:
```golang
	response, err := client.Do(request)
```